### PR TITLE
octal number just contain digits range from 0 to 7.

### PR DESCRIPTION
**Pull request by Turing-Chu**

### CODE REVIEW

File: syntax_and_semantics/literals/string.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Available escape sequences:
 
 Any other character following a backslash is interpreted as the character itself.
 
-A backslash followed by at most three digits denotes a code point written in octal:
+A backslash followed by at most three digits range from 0 to 7 denotes a code point written in octal:
```
Comment (mverzilli) [inline syntax_and_semantics/literals/string.md]: range from 0 to 7 => ranging from 0 to 7

**Comment by Turing-Chu** [comment]
thks for correcting syntax error. 😂

**Comment by straight-shoota** [comment]
Why closed?

**Comment by mverzilli** [comment]
Thank you!

**Pull request merged**